install-binary.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+
+# Shamelessly copied from https://github.com/technosophos/helm-template
+
+PROJECT_NAME="helm-update-config"
+PROJECT_GH="bluebosh/${PROJECT_NAME}"
+
+: "${HELM_PLUGIN_DIR:=\"$(helm home --debug=false)/plugins/helm-update-config\"}"
+
+# Convert the HELM_PLUGIN_DIR to unix if cygpath is
+# available. This is the case when using MSYS2 or Cygwin
+# on Windows where helm returns a Windows path but we
+# need a Unix path
+
+if type cygpath > /dev/null 2>&1; then
+  HELM_PLUGIN_DIR=$(cygpath -u "${HELM_PLUGIN_DIR}")
+fi
+
+if [[ $SKIP_BIN_INSTALL == "1" ]]; then
+  echo "Skipping binary install"
+  exit
+fi
+
+# initArch discovers the architecture for this system.
+initArch() {
+  ARCH=$(uname -m)
+  case $ARCH in
+    armv5*) ARCH="armv5";;
+    armv6*) ARCH="armv6";;
+    armv7*) ARCH="armv7";;
+    aarch64) ARCH="arm64";;
```

Comment:
Can these be in the else branch? Because we only support two ARCH: amd64 and i386.